README.md

```diff
@@ -65,8 +65,8 @@ This is the easiest way to get started for development with the permissioning Da
 
 2. The following additional environment variables are optional and can be used to permit accounts and nodes during initial contract deployment
   - `INITIAL_ADMIN_ACCOUNTS`: The admin account addresses. Comma-separated multiple addresses can be specified
-  - `INITIAL_WHITELISTED_ACCOUNTS`: The permitted account addresses. Comma-separated multiple addresses can be specified
-  - `INITIAL_WHITELISTED_NODES`: The enode URLs of permitted nodes. Comma-separated multiple nodes can be specified
+  - `INITIAL_ALLOWLISTED_ACCOUNTS`: The permitted account addresses. Comma-separated multiple addresses can be specified
```

Comment:
`ALLOWLISTED` or `PERMITTED` ?

Reply:
docs advice was not to use allowlist as a verb